website/assets/scss/common/_branding.scss

```
@@ -7,10 +7,8 @@
  * - Theme-dependent logo colors
  */
 
-@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap');
-
 .font-logo {
-  font-family: 'Quicksand', sans-serif;
+  font-family: 'Quicksand Variable', 'Quicksand', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Quicksand" (font-family-name-quotes)

(font-family-name-quotes)